src/utils/MultiInviter.js

```diff
@@ -104,7 +115,14 @@ export default class MultiInviter {
             return;
         }
 
-        inviteToRoom(this.roomId, addr).then(() => {
+        let doInvite;
+        if (this.groupId !== null) {
+            doInvite = MatrixClientPeg.get().inviteUserToGroup(this.groupId, addr);
```

Comment:
I was curious to know why we need `inviteToRoom` and it turns out that it's for inviting by email. I wonder if we're going to need to invite people to groups by email..

Reply:
Maybe eventually :)